custom_model_runner/datarobot_drum/drum/gpu_predictors/base.py

```diff
@@ -307,7 +308,12 @@ def _create_completions(self, messages, row_id=0):
         return completion_choices
 
     def predict_unstructured(self, data, **kwargs):
-        raise DrumCommonException("The unstructured target type is not supported")
+        model = None  # no artifacts are loaded, requests are proxied to a separate server
```

Comment:
Cool I have similar chances to this in my branch. Maybe we can consolidate efforts. I don't mind taking this over.

Reply:
See https://github.com/datarobot/datarobot-user-models/pull/1250